backend/aethel_db/urls.py

```diff
@@ -1,7 +1,8 @@
 from django.urls import path
-from aethel_db.views import AethelQueryView, AethelDetailView
+from aethel_db.views.AethelDetailView import AethelDetailView
```

Comment:
this naming scheme is technically fine but feels out of place in python. how about:

```
from aethel_db.views.detail import AethelDetailView 
from aethel_db.views.list import AethelListView
from aethel_db.views.status import ...
```

Reply:
I agree this feels better :)